src/Core/Engine/Physics/PhysicsWorld.hpp

```
@@ -0,0 +1,39 @@
+    void IntegrateVelocities(float dt);
+    void DetectCollisions();
+    void ResolveContacts(float dt);
+    void SyncTransforms();
+
+    void ContactAABB(PhysicsBody& PhysBodyA, PhysicsBody& PhysBodyB);
+
+private:
+
+    void HandleAABB(PhysicsBody& PhysBodyA, PhysicsBody& PhysBodyB);
+
+private:
```

Clang (14.0.6)

[warning] redundant access specifier has the same accessibility as the previous access specifier

(readability-redundant-access-specifiers)